develop/best_practices/Creating_unit_tests_in_protocols.md

```diff
@@ -0,0 +1,638 @@
+---
+uid: Creating_unit_tests_in_protocols
```

Comment:
This page is quite big, maybe it's better to make subpages corresponding to the blog post series: https://community.dataminer.services/unit-testing-data-driven-tests/
I think the article for Data-driven tests is also missing here.

Reply:
Done!